src/DGmethods/balancelaw.jl

```diff
@@ -37,6 +37,11 @@ function vars_gradient end
 function vars_diffusive end
 vars_integrals(::BalanceLaw, FT) = @vars()
 
+# additional stubs for default units
+function spatial_unit end
```

Comment:
Why don't we use 
```suggestion
function spacel_unit end
```
to construct `space_unit` and `time_unit`(rather than `temporal_unit` in parallel)?